artifact/image/image.go

```diff
@@ -19,14 +19,16 @@ import (
 )
 
 type Artifact struct {
-	image image.Image
-	cache cache.ArtifactCache
+	image           image.Image
+	cache           cache.ArtifactCache
+	skipDirectories []string
```

Comment:
I considered it again and now I feel `skipDirectories` is more `Inspect` stuff. What if we pass it as the argument of `Inspect`? We'll likely have more options for `Inspect`, so it might be better to provide `InspectOption`.
However, I haven't looked into what change we need. Let me know if we need any bigger change than the current implementation.